ProteGO/Common/Validator/ValidationType.swift

```diff
@@ -0,0 +1,16 @@
+enum ValidationType {
+    case phoneNumber
+    var regex: String {
+        switch self {
+        case .phoneNumber:
+            return "^\\+48[0-9]{9}$"
```

Comment:
Czy to jest wymaganie projektu, że tylko polskie numery telefonów są obsługiwane? -> Prefix country code w regex

Reply:
<img width="545" alt="Screenshot 2020-04-09 at 23 17 25" src="https://user-images.githubusercontent.com/23533452/78941649-611bbb80-7ab8-11ea-86c9-2bcc61c9d9f3.png">
 i 

`extension Constants {

    enum Networking {

        static let apiVersion = "1"

        static let phoneNumberPrefix = "+48"
    }
}`

Pozwalają mi twierdzić, że tak :)

Prefix telefonu nie jest edytowalny

Reply:
Tak, prefix jest zahardcodowany. Prośba o skorzystanie ze stałej w regexie

Reply:
Tak, prefix jest zahardcodowany. Prośba o skorzystanie ze stałej w regexie